src/platform/site-wide/sass/shame.scss

```diff
@@ -577,3 +577,7 @@ img[data-srcset] {
   display: block;
   clear: both;
 }
+
+.va-u-outline--none:focus {
```

Comment:
Is this something that should be updated on the component or design system css?

Reply:
Probably needs to be updated in the design system... I'll open a ticket.

Reply:
Actually, it's not a design system class (it doesn't start with `vads`)... but I do think we need to include some sort of class to remove outlines in foundation.

Reply:
See https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/325